core/config/v2/docs/core.toml

```diff
@@ -500,3 +500,17 @@ DSN = 'sentry-dsn' # Example
 Environment = 'my-custom-env' # Example
 # Release overrides the Sentry release to the given value. Otherwise uses the compiled-in version number.
 Release = 'v1.2.3' # Example
+
+
+# Insecure config family is only allowed in development builds.
+[Insecure]
+# **ADVANCED**
+# InsecureDevWebServer skips secure configuration for webserver AllowedHosts, SSL, etc.
+InsecureDevWebServer = false # Default
+# InsecureOCRDevelopmentMode run OCR in development mode.
+InsecureOCRDevelopmentMode = false # Default
+# InsecureInfiniteDepthQueries skips graphql query depth limit checks.
+InsecureInfiniteDepthQueries = false # Default
+# InsecureDisableRateLimiting skips ratelimiting on asset requests.
+InsecureDisableRateLimiting = false # Default
```

Comment:
Drop the prefixes since we're alraedy scoped to the `Insecure` table?
```suggestion
# DisableRateLimiting skips ratelimiting on asset requests.
DisableRateLimiting = false # Default
```

Reply:
skipped